src/config/settings.py

```diff
@@ -376,6 +376,15 @@ class AtIndex:
             9  = [Keys.alt_or_cmd() + "+9"]
             10 = [Keys.alt_or_cmd() + "+0"]
 
+        class Direct:
+            # Switch to a specific room with a keybinding.
+            # Each property is a list of keybinds for some room ID.
+            # Optionally, the account can be specified
+            # by using a space separator (see example).
+            # Unlimited entries can be added.
+            "!uxYVyZEASzrDCXMnNO:matrix.org" = []
```

Comment:
The default code shouldn't point to a real room but be a clear example, e;g. `!roomID:example.org`